docs/release-1.2/existing-rhdh.md

```diff
@@ -64,121 +91,156 @@ metadata:
-    pluginConfig:
-      dynamicPlugins:
-        frontend:
-          redhat.plugin-notifications:
-            dynamicRoutes:
-              - importName: NotificationsPage
-                menuItem:
-                  config:
-                    props:
-                      titleCounterEnabled: true
-                      webNotificationsEnabled: false
-                  importName: NotificationsSidebarItem
-                path: /notifications
-  - disabled: false
-    package: "@redhat/plugin-signals-dynamic@1.2.0"
-    integrity: sha512-5tbZyRob0JDdrI97HXb7JqFIzNho1l7JuIkob66J+ZMAPCit+pjN1CUuPbpcglKyyIzULxq63jMBWONxcqNSXw==
-    pluginConfig:
-      dynamicPlugins:
-        frontend:
-          redhat.plugin-signals: {}
-  - disabled: false
-    package: "@redhat/plugin-notifications-backend-dynamic@1.2.0"
-    integrity: sha512-pCFB/jZIG/Ip1wp67G0ZDJPp63E+aw66TX1rPiuSAbGSn+Mcnl8g+XlHLOMMTz+NPloHwj2/Tp4fSf59w/IOSw==
-  - disabled: false
-    package: "@redhat/plugin-signals-backend-dynamic@1.2.0"
-    integrity: sha512-DIISzxtjeJ4a9mX3TLcuGcavRHbCtQ5b52wHn+9+uENUL2IDbFoqmB4/9BQASaKIUSFkRKLYpc5doIkrnTVyrA==
+      - disabled: false
+        package: "@redhat/plugin-notifications-backend-module-email-dynamic@1.2.0"
+        integrity: sha512-dtmliahV5+xtqvwdxP2jvyzd5oXTbv6lvS3c9nR8suqxTullxxj0GFg1uU2SQ2uKBQWhOz8YhSmrRwxxLa9Zqg==
+        pluginConfig:
+          notifications:
+            processors:
+              email:
+                transportConfig:
+                  transport: smtp
+                  hostname: ${NOTIFICATIONS_EMAIL_HOSTNAME}   # Use value or make variable accessible to Backstage
+                  port: 587
+                  secure: false
+                  username: ${NOTIFICATIONS_EMAIL_USERNAME}   # Use value or make variable accessible to Backstage
+                  password: ${NOTIFICATIONS_EMAIL_PASSWORD}   # Use value or make variable accessible to Backstage
+                sender: sender@mycompany.com
+                replyTo: no-reply@mycompany.com
+                broadcastConfig:
+                  receiver: "none"
+                concurrencyLimit: 10
+                cache:
+                  ttl:
+                    days: 1
 ```
 
-For the `*-app-config` ConfigMap add the database configuration if isn't already provided. It is required for the notifications plugin:
+### app-config ConfigMap
+This ConfigMap is used for configuring backstage. Please add/modify to include the following:
+- `${BACKEND_SECRET}` A static access token to enable the workflows to send notifications to RHDH or to invoke scaffolder actions (Or a different method based on this [doc](https://backstage.io/docs/auth/service-to-service-auth/)).
+- A static access token can be generated with this command `node -p 'require("crypto").randomBytes(24).toString("base64")'`
+- `${RHDH_ROUTE}` can be determined by running `oc get route -A -l app.kubernetes.io/name=backstage`
+- Define csp and cors
```

Comment:
maybe adding something like "as described in the example below" can clarify this step